Pipfile

```
@@ -19,9 +19,9 @@ pytest-trio = "*"
 
 [packages]
 django = "~=2.2"
-stomp-py = "*"
-request-id-django-log = "*"
-tenacity = "*"
+stomp-py = "~=4.1"
```

```suggestion
"stomp.py" = "~=4.1.22"
```

If you install as `stomp-py`(pay attention to the dash instead of the dot), I think you end up installing something else or an unexpected version. The `Pipfile.lock` shows `stomp-py version 4.1.15` instead of `4.1.22` which is the expected for the use case `~=4.1`.

I'm still unsure about leaving it like that.